src/tasks/models/tasksManager.ts

```diff
@@ -63,7 +64,7 @@ export class TasksManager {
       artifactCRS: EPSGDATA[4326].code,
       description: params.description,
       keywords: jobParameters.keywords,
-      status: params.status,
+      status: convertToUnifiedTaskStatus(params.status),
```

Comment:
Check isCleaned in job to return appropriate status, ask @syncush if we need to remove artifact URI as well

Reply:
E2E check that response cahanges for cleaned artifact

Reply:
Check for precentage change while task is running